Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/SingleFileTransfer.cs

```diff
@@ -441,7 +442,7 @@ private void TryToRemoveLocalTempFile(string filePath)
         }
```

Comment:
https://github.com/FrendsPlatform/Frends.SFTP/blob/4b775a08a2df514e8b4c67800d9a4757599f9c04/Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/SingleFileTransfer.cs#L433
This can be a one-line if-statement.